perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -173,6 +173,28 @@ public Group call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Create union of two groups, where "operandGroup" is technically set as subgroup of "resultGroup".
+	 * Members from "operandGroup" are added to "resultGroup" as INDIRECT members. Union is honored also
+	 * in all group member changing operations.
+	 *
+	 * @param resultGroup int <code>id</code> of Group to have included "operandGroup"
+	 * @param operandGroup int <code>id</code> of Group to be included into "resultGroup"
+	 * @return Group Result group
+	 */
+	moveGroup {
+
+		@Override
+		public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
+			ac.stateChangingCheck();
+
+			ac.getGroupsManager().moveGroup(ac.getSession(),
+					ac.getGroupById(parms.readInt("destinationGroup")),
```

Comment:
Unlike in perun-core, there is no possibility to provide ``null`` for ``destinationGroup``, since it always retrieve group by provided ID. Please make ``destinationGroup`` param optional (check it's presence) to allow moving groups to top-level.